indico/modules/events/persons/placeholders.py

```diff
@@ -63,6 +64,27 @@ def render(cls, person, event, **kwargs):
                                                                           title=event.title)
 
 
+class EventContributionsPlaceholder(ParametrizedPlaceholder):
+    name = 'event_contributions'
+    description = _("Event contributions, optional: One where speaker")
```

Comment:
Instead of documenting the optional arg here, better do this (and also enable `param_restricted = True`):

```python
    @classmethod
    def iter_param_info(cls, person, event, **kwargs):
        yield None, _("The person's contributions")
        yield 'onlyspeakers', _("The person's contributions where they are a speaker")
```

You can most likely ditch the default `description` arg then.